vendor/github.com/aead/poly1305/poly1305_ref.go

```diff
@@ -34,7 +32,8 @@ func Sum(msg []byte, key [32]byte) [TagSize]byte {
 		msg = msg[n:]
 	}
 	if len(msg) > 0 {
-		off = copy(block[:], msg)
+		var block [TagSize]byte
```

Comment:
If len(msg) is a multiple of 16 bytes (and this the remainder = 0, there is no update() using the finalBlock?
Is that correct behaviour?